torque-core/src/test/kotlin/com/workday/torque/ModuleTestParserSpec.kt

```diff
@@ -98,7 +98,7 @@ class ModuleTestParserSpec : Spek(
 			val args = Args().apply {
 				testApkPaths = listOf(testApkPath)
 				appApkPath = targetApkPath
-				includedAnnotations = listOf("MediumTest")
+				includedAnnotations = listOf("androidx.test.filters.MediumTest")
```

Comment:
Do all annotations passed to Torque need to be fully qualified now? If so, we should make that explicit in all of the documentation including `Args`, `TorqueRunTask`, and also update the sample in the README

Reply:
Yes dex test parser returns full qualified annotations now so we need to match on the full path. I'll update the documentation